e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-supply-contract-validation.spec.js

```diff
@@ -322,4 +322,31 @@ context('about-supply-contract', () => {
     partials.errorSummary.errorSummaryLinks().should('have.length', 12);
     contractAboutSupplier.expectError('Enter a valid Companies House registration number');
   });
+
+  it('A maker picks up a deal in status=Draft, misses mandatory field then trigger missed validation error.', () => {
+    cy.login(BANK1_MAKER1);
+
+    contractAboutSupplier.visit(deal);
+    // click through
+    contractAboutSupplier.supplierType().select('Exporter');
+    contractAboutSupplier.supplierName().type('test');
+    contractAboutSupplier.supplierAddress().line1().type('test');
+    contractAboutSupplier.supplierAddress().town().type('test');
+    contractAboutSupplier.supplierCorrespondenceAddressDifferent().click();
+    contractAboutSupplier.supplierCorrespondenceAddress().line1().type('test');
+    contractAboutSupplier.supplierCorrespondenceAddress().town().type('test');
+    contractAboutSupplier.supplierCorrespondenceAddress().country().select('GBR');
+    contractAboutSupplier.industrySector().select('Education');
+    contractAboutSupplier.industryClass().select('Cultural education');
+    contractAboutSupplier.smeTypeMicro().click();
+    contractAboutSupplier.supplyContractDescription().type('test');
+    contractAboutSupplier.legallyDistinct().click();
+    contractAboutSupplier.nextPage().click();
+    // re-visit supplier page
+    partials.taskListHeader.itemLink('supplier-and-counter-indemnifier/guarantor').click();
+    // should display missed validation errors
+    partials.errorSummary.errorSummaryLinks().should('have.length', 6);
```

Comment:
why 6?

Reply:
It returns 6 summary links hence comparing with 6

Reply:
Added comment